### ISSUE TITLE
Visual Label not provided for "Select a duration" Combo box: A11y_Aspire Dashboard_Metrics_.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/aspire#10159]
### GitHubTags
#A11ySev2; #A11yTCS;#DesktopWeb; #Win32;#.NET Project system for CSharp and VB;#BM_AspireDashboard_Web_June25;#Win32;#WCAG3.3.2;#Labels or Instructions;#A11yMAS;

### Environment Details:
Application Name: Aspire Dashboard
Edition Windows 11 Enterprise
Version 24H2
OS build 26100.4351

### Repro Steps

1. Open Visual studio.
2. Open a aspire project and run it.
3. A new window will open in the chromium edge,
4. TAB to "Metrics" tab item and hit ENTER key.
5. Verify that Visual Label not provided for "Select a duration" Combo box.

### Actual Result:
Visual Label not provided for "Select a duration" Combo box.

### Expected Result:
The input field should have a clear and descriptive visible label (e.g., "Select a duration") to guide users.

### User Impact:
Cognitive disabilities users or those who relying on assistive technologies may not understand the purpose of the field.

### Attachment:

![Image](https://github.com/user-attachments/assets/fa509a0a-6aae-40f9-a4e7-87f1210cfb20)
